prow/istio-postsubmit.sh

```diff
@@ -48,4 +48,4 @@ time ISTIO_DOCKER_HUB="${HUB}" make push HUB="${HUB}" TAG="${TAG}"
 
 cd "$ROOT"
 OUTDIR="$(mktemp -d /tmp/outdir.XXXX)"
-time (./release/cloud_builder.sh -p istio-release-pipeline-data/daily-build/postsubmit/"${GIT_SHA}" -q istio-release -t "${GIT_SHA}"  -o "${OUTDIR}" || echo "not failing make for now")
+time (./release/cloud_builder.sh -h "gcr.io/istio-testing" -t "${GIT_SHA}"  -o "${OUTDIR}" || echo "not failing make istio-archive for now")
```

Comment:
why is the trailing echo needed to prevent failure?

Reply:
this will be moved to a different job, currently zip is not installed due to move to pod-utils

Reply:
Not sure I understand. cloud_builder.sh is currently failing due to not having zip installed? If so, why?